relay-server/src/services/store.rs

```diff
@@ -695,20 +734,19 @@ impl StoreService {
         retention: u16,
     ) -> Result<(), StoreError> {
         let payload = item.payload();
+        let encodings = self.global_config.current().options.metric_bucket_encodings;
 
         for bucket in serde_json::from_slice::<Vec<Bucket>>(&payload).unwrap_or_default() {
-            self.send_metric_message(
-                org_id,
-                MetricKafkaMessage {
-                    org_id,
-                    project_id,
-                    name: bucket.name.as_str(),
-                    value: (&bucket.value).into(),
-                    timestamp: bucket.timestamp,
-                    tags: &bucket.tags,
-                    retention_days: retention,
-                },
-            )?;
+            let view = BucketView::new(&bucket);
+
+            let namespace = MetricResourceIdentifier::parse(view.name())
```

Comment:
Should we pass the parsed MRI from `handle_store_message` instead of parsing again?